### PR TITLE
Use Gen3 schemas in transforms

### PR DESCRIPTION
**Pull request by adamstruck**
Builds off #329

### CODE REVIEW

**Comment by adamstruck** [comment]
From review on 6/27:

SCHEMA TODO:
- change chromosome definition 
  - no “chr” prefix
- remove “GENE_TPM”
- bring over “descriptions” from gdc schemas
- check “to_one” “to_many” relationships
- rename Callset to SomaticCallset

ETL TODO:
- no multiplexed edge files (i.e. they should be named callset_alleles_allele.Edge.json.gz)
- remove “_id”
- submitter_id != gid; case by case depending on vertex type
- add id property to hold gid; remove from props during emit and validation
- fix cell lines case <-> project relationship
  - create case for each cell line
  - create edge between equivalent cases

File: src/bmeg/__init__.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+            '_gid_cls': types.new_class("{}GID".format(name), (str,), {}),
+            'make_gid': classmethod(cast_gid(gid_factory))
+        }
+    )
+    globals()[name] = cls
+    __all__.append(name)
+
+for k, schema in _schema.schema.items():
+    for link in schema['links']:
+        # TODO: handle link subgroup?
```
Comment (adamstruck) [inline src/bmeg/__init__.py]: @bwalsh did you mention that you had code that demonstrates how to handle link subgroups?

**Comment by adamstruck** [comment]
Missing vertex summary from running bloom filter:
```
{
  "Transcript": 129,
  "Gene": 8,
  "Aliquot": 304,
  "GeneOntologyTerm": 11,
  "Protein": 15195,
  "PfamFamily": 3
}
```

Seems to be in line with our previous releases.

**Comment by kellrott** [comment]
👍

**Pull request merged**